components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/util/Utils.java

```diff
@@ -246,22 +247,7 @@ public static String getAuthenticatedUsername() {
      */
     public static String getUserId() {
 
-        String userId = PrivilegedCarbonContext.getThreadLocalCarbonContext().getUserId();
-        /*
-            The federated users with organization management permissions do not have user id set in the carbon
-            context but the user id is set against the username. Therefore the user is resolved by the user id
-            information saved in the username.
-         */
-        if (userId == null && PrivilegedCarbonContext.getThreadLocalCarbonContext().getUsername() != null) {
-            try {
-                userId = organizationUserResidentResolverService.resolveUserFromResidentOrganization(null,
-                        PrivilegedCarbonContext.getThreadLocalCarbonContext().getUsername(),
-                        getOrganizationId()).map(User::getUserID).orElse(null);
-            } catch (OrganizationManagementException e) {
-                LOG.debug("Authenticated user's id could not be resolved.", e);
-            }
-        }
```

Comment:
This part is not more required due to the user-id is properly set in the carbon context by this logic for the users federated via organizationSSO.
https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/252/files#diff-604290ac058e0ac40d2bd89a168f3e62a58fcbd753e58ddadf8066bfaed56701R127-R131